Source/FlavoredNeutrinoContainer.H

```diff
@@ -94,6 +96,11 @@ public:
         return attribute_names;
     }
 
+    void copyParticles(const FlavoredNeutrinoContainer& other, bool local=false){
+        amrex::ParticleContainer<PIdx::nattribs, 0, 0, 0>::copyParticles(other, local);
+        this->Vvac_max = other.Vvac_max;
```

Comment:
I think making Vvac_max a static member instead would let us not have to worry about keeping it updated between particle containers.

Reply:
That's a good idea! I'll give it a shot.

Reply:
so to make a static member variable that doesn't need some wonky declaration in the cpp files, it has to have the inline  attribute, which is only available for static members when using -std=c++17. But it is available as of gcc 7

Reply:
oh cool! That sounds good to me, AMReX requires at least c++14, so that doesn't seem like a stretch, and I think all the supercomputers we're using have a more recent gcc version than that.